### ISSUE TITLE
scrollable-region-focusable should ignore tiny scroll regions

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#4826]
**Expetation**: This shouldn't fail the `scrollable-region-focusable` rule:

```html
<button>
  <span aria-hidden="true">☕</span>
  <span style="position: absolute; width: 1px; height: 1px; overflow-x: hidden">
    Coffee
  </span>
</button>
```

The scroll region is only 1px wide, it can't ever show even a single letter in full. There's no way this element could possibly be a scrollable element.
